Website/src/components/Headers/HomeHeader.jsx

```diff
@@ -24,7 +24,7 @@ function IndexHeader() {
 
   return (
     <>
-      <div className="page-header clear-filter" filter-color="blue">
```

Comment:
Please move all styles to the tkot.scss file under an appropriate CSS class name, then reference the CSS class name in the component.

E.g. Change from:
HomeHeader.jsx
`<div className="page-header clear-filter" style={{backgroundColor: 'rgb(23,27,93, 0.8)'}}>`

To:
tkot.scss
`.home-header-page-header {
  backgroundColor: rgb(23,27,93, 0.8);
}`
HomeHeader.jsx
`<div className="page-header clear-filter home-header-page-header">`